src/EditorFeatures/Core/Tagging/TaggerContext.cs

```diff
@@ -69,7 +70,6 @@ internal sealed class TaggerContext<TTag> where TTag : ITag
         this.SpansToTag = spansToTag;
         this.CaretPosition = caretPosition;
 
-        _spansTagged = spansToTag.SelectAsArray(ds => ds.SnapshotSpan);
```

Comment:
>_spansTagged = spansToTag.SelectAsArray(ds => ds.SnapshotSpan); [](http://example.com/codeflow?start=8&length=63)

I assume this was remove because the caller calls SetSpansTagged after creating?

Reply:
i revised this (and added docs).  LMK if it's clearer now.

Reply:
Yes, thank you!